SayTheirNames/Source/Controller/Home/CollectionViewDataSource/PersonCollectionViewDataSource.swift

```diff
@@ -45,11 +45,14 @@ final class PersonCollectionViewDataSourceHelper {
             cell.configure(with: person)
             cell.accessibilityIdentifier = "peopleCell\(indexPath.item)"
             cell.isAccessibilityElement = true
+            cell.accessibilityNavigationStyle = .automatic
+            cell.accessibilityLabel = "\(person.fullName), tap to learn more"
```

Comment:
Can you remove "tap to learn more" from accessibility label
I think HIG recommends not to tell users how to interact with label manually. Instead put "Learn more" in accessibility hint.
https://developer.apple.com/documentation/objectivec/nsobject/1615093-accessibilityhint#:~:text=An%20accessibility%20hint%20helps%20users,obvious%20from%20the%20accessibility%20label.

Reply:
Good point! Done